docs/make.jl

```diff
@@ -1,4 +1,9 @@
-using Documenter, Skyrmions3D
+using Documenter, Skyrmions3D, DocumenterCitations
+
+bib = CitationBibliography(
+    joinpath(@__DIR__, "src/ref.bib");
```

Comment:
are you able to change this now to `"paper/paper.bib"` because the paper PR has been merged?